src/main/java/org/example/catcafereservation/ReservationController.java

```diff
@@ -21,4 +22,29 @@ public ResponseEntity<Reservation> getReservation(@PathVariable String reservati
         Reservation reservation = reservationService.findByReservationNumber(reservationNumber);
         return ResponseEntity.ok(reservation);
     }
+
+    @PostMapping("/")
+    public ResponseEntity<ReservationResponse> insert(@RequestBody ReservationRequest reservationRequest, UriComponentsBuilder uriBuilder) {
+        Reservation reservation = reservationService.insert(
+                reservationRequest.getName(),
+                reservationRequest.getReservationDate(),
+                reservationRequest.getReservationTime(),
+                reservationRequest.getEmail(),
+                reservationRequest.getPhone()
+        );
+
+        String reservationNumber = reservationService.generateReservationNumber(reservation.getReservationDate(), reservation.getId());
+
+        URI location = uriBuilder.path("/reservations/{reservation_number}").buildAndExpand(reservationNumber).toUri();
```

Comment:
細かいですが、` @GetMapping("/{reservationNumber}")`と表記を合わせています。
スネークケースにしないと動かなさそうですか？？

```suggestion
        URI location = uriBuilder.path("/reservations/{reservationNumber}").buildAndExpand(reservationNumber).toUri();
```

Reply:
気づいていただき有難う御座います！
キャメルケースにて、問題なく動くことを確認いたしました。

- コミットハッシュ： f0448cdb00d93249c83951da730cf70e9a785aeb

以下は修正後の登録成功画面です。
![image](https://github.com/Ema-Sakai/Assignment-10/assets/166620990/c77ed35b-4947-44ab-a747-b619b7812753)